src/InvoiceBundle/Resources/views/Pdf/invoice.html.twig

```diff
@@ -10,7 +10,7 @@
 <html>
 <head>
     <style type="text/css">
-        {{ file('css/pdf.css') }}
+        {{ asset('static/pdf.css') }}
```

Comment:
I don't think this will work. The `asset` function just returns a URL to the stylesheet, but `file` includes the contents inline.
We need to output the content of the stylesheet here since the PDF processor uses the inline styles when generating the PDF file

Reply:
I was getting an Exception here since the file was not present where it was expected. One question would be if file can fetch the url. 

for me after `yarn build` the css directory is empty. Any idea why?

Reply:
Thanks for reviewing this. Regarding:

> I think an alternative option is to have a field when creating an invoice, where you can then specify your own invoice number to use. If the field is empty, it then falls back to the auto-generated id.
This would then also pave the way for the long-term solution that you suggested, where we can then have generators that can then just pre-fill the field.

I will need some hints if i will try to implement that. Honestly don't have an idea yet how to implement it, sorry no expert in Symfony.

Reply:
> for me after yarn build the css directory is empty. Any idea why?

I think this wasn't updated after the project moved to using Webpack. Can you please open a new issue for this, then I can investigate and fix this separate from the invoice id.